lib/LaTeXML/Util/Pathname.pm

```diff
@@ -373,9 +373,9 @@ sub candidate_pathnames {
           qr/^\Q$name\E\Q$ext\E$/]); } }
   # Now, combine; precedence to leading directories.
   foreach my $dir (@dirs) {
-    opendir(DIR, $dir) or next;
-    my @dir_files = readdir(DIR);
-    closedir(DIR);
+    opendir(my $dir_handle, $dir) or next;
+    my @dir_files = readdir($dir_handle);
+    closedir($dir_handle);
```

Comment:
this change was a recommendation (severity 5) by perl critic - using a lexical var name.